test/utils/utils.go

```diff
@@ -88,6 +88,56 @@ func WaitForPods(t *testing.T, f *test.Framework, namespace string, labels map[s
 	return pods, nil
 }
 
+func WaitForRollingRestart(t *testing.T, f *test.Framework, namespace string, labels map[string]string, excludePods []string, retryInterval, timeout time.Duration) (*corev1.PodList, error) {
```

Comment:
technically we aren't waiting for a specific sort of restart, we are waiting for new pods to be rolled out (by k8s)

Reply:
This function waits for all new pods except the ones in `excludedPods` and this is the difference to the existing `WaitForPods()` function.

Reply:
right, maybe something like "WaitForRolloutComplete" ?